biome.json

```diff
@@ -0,0 +1,31 @@
+{
+  "formatter": {
+    "enabled": true,
+    "formatWithErrors": false,
+    "ignore": [],
+    "attributePosition": "auto",
+    "indentStyle": "space",
```

Comment:
Spaces won. GH is not happy with tabs, and that's the truth

@rafaellvs skinny bud